inc/class-wp-bootstrap-navwalker.php

```diff
@@ -407,7 +407,7 @@ public static function fallback( $args ) {
 		 *
 		 * @return array  $classes         a maybe modified array of classnames.
 		 */
-		private function seporate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
+		private function separate_linkmods_and_icons_from_classes( $classes, &$linkmod_classes, &$icon_classes, $depth ) {
```

Comment:
I think this is the only change I'd like to see removed from this PR. If we want to update the name of this function, I think we'd need to still keep the older one and deprecate it to be safe, but I think we'd want that in a separate PR.

Reply:
Just realizing we already had this discussion about this being a private function. So to be clear - we see no reason to add the old function with a deprecation notice, correct?

Reply:
Yes, correct.

Reply:
Noting here that once this is merged to understrap I would be more than happy to merge the same changeset to the navwalker repo as well. I have partially stepped away from that project, but I trust that the changes here are being tested and done for the best of reasons so would want to roll that back to the origin repo as well :)